Source/Core/Common/ChunkFile.h

```diff
@@ -235,7 +235,7 @@ class PointerWrap
 
 		while (true)
 		{
-			u8 shouldExist = (list_cur ? 1 : 0);
+			u8 shouldExist = !!list_cur;
```

Comment:
From IRC: "[...] i don't really mind either way since you brought enough arguments that it should be ok to do so [...]"